pkg/apk/apk.go

```diff
@@ -126,6 +132,64 @@ func (a *APK) Initialize(ic *types.ImageConfiguration) error {
 	return nil
 }
 
+// AdditionalTags is a helper function used in conjunction with the --package-version-tag flag
+// If --package-version-tag is set to a package name (e.g. go), then this function
+// returns a list of all images that should be published with the associated version of that package tagged (e.g. 1.18)
+func AdditionalTags(opts options.Options) ([]string, error) {
+	if opts.PackageVersionTag == "" {
+		return nil, nil
+	}
+	dbPath := filepath.Join(opts.WorkDir, "lib/apk/db/installed")
```

Comment:
There is already code to parse the IDB into package objects.  See https://github.com/chainguard-dev/apko/blob/main/pkg/sbom/sbom.go#L88-L97.

This is probably more robust than parsing the `V:` field out by hand, since the order of fields is not formally defined.

Reply:
Thanks, this is much better!